test/package.json

```diff
@@ -4,8 +4,12 @@
   "private": true,
   "dependencies": {
     "aws-sdk": "^2.102.0",
+    "axios": "^0.20.0",
     "chai": "^3.5.0",
+    "cli-progress": "^3.8.2",
+    "csv-parse": "^4.12.0",
```

Comment:
I don't see where we're using this. Do we need it?

Reply:
good catch!